plugins/micromega/zify.ml

```diff
@@ -494,6 +494,12 @@ let mkapp = lazy (zify "mkapp")
 let mkapp0 = lazy (zify "mkapp0")
 let mkdp = lazy (zify "mkinjterm")
 let eq_refl = lazy (zify "eq_refl")
+
+let eq =
+  lazy
+    (EConstr.of_constr
+       (UnivGen.constr_of_monomorphic_global (Coqlib.lib_ref "core.eq.type")))
```

Comment:
This is not future proof, as it relies on deprecated code that will misbehave on polymorphic equalities.